src/restApi/util.ts

```diff
@@ -1,25 +1,32 @@
+type ConcreteType<Type> = {
```

Comment:
This is supported OOTB via the `Required` type

https://github.com/Microsoft/TypeScript/blob/a3c22683ab7462dc71935b3219098a3a3c98c9ea/lib/lib.es5.d.ts#L1418-L1423